src/test/regress/sql/multi_extension.sql

```diff
@@ -175,3 +175,131 @@ ALTER EXTENSION citus UPDATE;
 
 -- if cache is invalidated succesfull, this \d should work without any problem
 \d
+
+\c - - - :master_port
+
+CREATE VIEW table_fkeys AS
+SELECT r.conname AS "Constraint",
+       pg_catalog.pg_get_constraintdef(r.oid, true) AS "Definition",
+       conrelid AS "relid"
+FROM pg_catalog.pg_constraint r
+WHERE r.contype = 'f';
```

Comment:
FWIW I based all these off of the commands that `psql` prints when using `\d` commands with `ECHO_HIDDEN` set to on. Do those change that frequently?